day6/index.js

```diff
@@ -0,0 +1,45 @@
+$(() => {
+  const dict = ['apprehensive', 'barricade', 'condition', 'dictionary', 'fragile', 'grueling', 'habitation', 'illuminate',
+                'jostle', 'kindle', 'luminous', 'materialize', 'outlandish', 'plenteous', 'question', 'restaurant',
+                'seperate', 'turtle','umbrella','versatile','weather','xray','yoyo','zepalz']
+
+  const randomWord = () => {
+    let random = dict[Math.floor((Math.random() * (dict.length)))]
+    for (var i = 7; i > 0; i--) {
+      $('#used').append(`<span id="used${i}">_ </span>`)
+    }
+    for (var i = 0; i < random.length; i++) {
+      if (random.includes(random[i]))
+        $('#answer').append(`<span class="ans${random.indexOf(random[i])}">_ </span>`)
```

Comment:
`random.indexOf(random[i])` is redundant, why not just use `i` ?
You can simply remove the `if` condition and just do
```
for (var i = 0; i < random.length; i++) {
  $('#answer').append(`<span class="ans${i}">_ </span>`)
}
```